debezium-connector-postgres/src/main/java/io/debezium/connector/postgresql/connection/wal2json/Wal2JsonReplicationMessage.java

```diff
@@ -139,110 +150,213 @@ public String getName() {
+                    return new PGmoney(rawValue.asString()).val;
                 } catch (final SQLException e) {
                     LOGGER.error("Failed to parse money {}, {}", rawValue.asString(), e);
                     throw new ConnectException(e);
                 }
-            case "interval":
+            case "path":
                 try {
-                    return rawValue.isNotNull() ? new PGInterval(rawValue.asString()) : null;
+                    return new PGpath(rawValue.asString());
                 } catch (final SQLException e) {
                     LOGGER.error("Failed to parse point {}, {}", rawValue.asString(), e);
                     throw new ConnectException(e);
                 }
-            case "_int2":
-            case "_int4":
-            case "_int8":
-            case "_text":
-            case "_numeric":
-            case "_float4":
-            case "_float8":
-            case "_bool":
-            case "_date":
-            case "_time":
-            case "_timetz":
-            case "_timestamp":
-            case "_timestamptz":
-            case "_bytea":
-            case "_varchar":
-            case "_oid":
-            case "_bpchar":
-            case "_money":
-            case "_name":
-            case "_interval":
-            case "_char":
-            case "_varbit":
-            case "_uuid":
-            case "_xml":
-            case "_point":
-            case "_jsonb":
-            case "_json":
-            case "_ref_cursor":
+            case "point":
                 try {
-                    if (rawValue.isNull()) {
-                        return null;
-                    }
-                    final String dataString = rawValue.asString();
-                    PgArray arrayData = new PgArray(connection.get(), Oid.valueOf(columnType.substring(1) + "_array"), dataString);
-                    Object deserializedArray = arrayData.getArray();
-                    return Arrays.asList((Object[])deserializedArray);
+                    return new PGpoint(rawValue.asString());
+                } catch (final SQLException e) {
+                    LOGGER.error("Failed to parse point {}, {}", rawValue.asString(), e);
+                    throw new ConnectException(e);
                 }
-                catch (SQLException e) {
-                    LOGGER.warn("Unexpected exception trying to process PgArray column '{}'", columnName, e);
+            case "polygon":
+                try {
+                    return new PGpolygon(rawValue.asString());
+                } catch (final SQLException e) {
+                    LOGGER.error("Failed to parse point {}, {}", rawValue.asString(), e);
+                    throw new ConnectException(e);
                 }
-                return null;
+
+            // catch-all for other known/builtin PG types
+            // TODO: improve with more specific/useful classes here?
+            case "bit":
```

Comment:
By listing them here the log message below won't be printed. I'd say it should be enough to have just the `default` clause and then leave the method with the `return` below.

Reply:
these are types specifically builtin to PG which is why I listed them, so seems we should list them. But can fall-through to default easily enough.